modules/testing/pages/build/snyk.adoc

```diff
@@ -63,4 +75,5 @@ This action will make use of the `snyk ignore` command.
 [role="_additional-resources"]
 == Additional resources
 
-For more information about Snyk, see link:https://snyk.io/product/snyk-code/[the Snyk website].
+* link:https://spaces.redhat.com/spaces/PRODSEC/pages/553454167/Integrating+Snyk+Code+Token+with+Konflux+Pipeline[Integrating Snyk Code Token with Konflux Pipeline]
```

Comment:
Same here, we should either remove this link or move it to downstream documentation.